Client/Frontend/Strings.swift

```diff
@@ -4701,6 +4706,26 @@ extension String {
         tableName: "TabLocation",
         value: "Secure connection. Enhanced Tracking Protection is off.",
         comment: "Accessibility label for the security icon in url bar")
+    public static let TabLocationLockButtonLargeContentTitle = MZLocalizedString(
+        key: "TabLocation.LockButton.LargeContentTitle.v122",
+        tableName: "TabLocation",
+        value: "Tracking Protection",
+        comment: "Large content title for the lock button.  This title is displayed when using accessible font sizes is enabled")
```

Comment:
nit, there's an extra space in all the comments
```suggestion
        comment: "Large content title for the lock button. This title is displayed when using accessible font sizes is enabled")
```